src/components/PurchaseOrder/POInvoices/constants.js

```diff
@@ -0,0 +1,30 @@
+import { FormattedMessage } from 'react-intl';
+
+export const COLUMN_INVOICE_DATE = 'invoiceDate';
```

Comment:
Maybe it makes sense to create a field names map for all columns and use them instead of "magic" strings, what do you think?